source/includes/odata/odata_basics.md

```diff
@@ -522,67 +668,79 @@ followed by the `asc` and `desc` keywords to specify the sort order (default is
 
 ## Specific record
 
+> Get the record with the recordid `c8f5d3e8e38ea441d9c4bfae5809655a58d4c2e8`
+
 ```http
-GET https://examples.opendatasoft.com/api/odata/baby_names_nc_2013(efc3e55da1dd591ba0c2bd42f0b0719e330f738f) HTTP/1.1
+GET https://documentation-resources.opendatasoft.com/api/odata/doc-geonames-cities-5000(c8f5d3e8e38ea441d9c4bfae5809655a58d4c2e8) HTTP/1.1
 ```
 
 ```json
 {
-    "@odata.context": "https://examples.opendatasoft.com/api/odata/$metadata#baby_names_nc_2013/$entity",
-    "recordid": "efc3e55da1dd591ba0c2bd42f0b0719e330f738f",
-    "gender": "M",
-    "state": "NC",
-    "number": 79,
-    "name": "Caden",
-    "year": "2013"
+    "@odata.context": "https://documentation-resources.opendatasoft.com/api/odata/$metadata#doc-geonames-cities-5000/$entity",
+    "recordid": "c8f5d3e8e38ea441d9c4bfae5809655a58d4c2e8",
+    "geonameid": "3165524",
+    "name": "Turin",
+    "asciiname": "Turin",
+    "alternatenames": "Augusta Taurinorum,Julia Augusta Taurinorum,Lungsod ng Turino,TRN,Tori,Torin,Torino,Torinu,Torí,Tueri,Turen,Turijn,Turim,Turin,Turina,Turinas,Turino,Turinu,Turyn,Turén,Turìn,Turín,Turīna,Tórínó,Türì,dou ling,tolino,torino,tu rin,tulin,turin,turina,twryn,twrynw,Τορίνο,Торино,Турин,Турын,Թուրին,טורין,טורינו,تورينو,تورین,टोरीनो,तोरिनो,তুরিন,துரின்,ตูริน,ཊུ་རིན།,ტურინი,トリノ,都灵,토리노,투린",
+    "feature_class": "P",
+    "feature_code": "PPLA",
+    "country_code": "IT",
+    "admin1_code": "12",
+    "admin2_code": "TO",
+    "admin3_code": "001272",
+    "admin4_code": "0012721",
+    "population": 870456,
+    "elevation": "239",
+    "dem": 245,
+    "timezone": "Europe/Rome",
+    "modification_date": "2019-09-05T00:00:00+00:00",
+    "geo_point_2d": {
+        "latitude": 45.07049,
+        "longitude": 7.68682
+    }
 }
 ```
 
-To access a specific record, its record id surrounded by parenthesis, can be appended to the dataset address.
+To access a specific record, append its record id surrounded by parentheses to the dataset address.
```

Comment:
Shouldn't it be consistent : either recordid or record id ?

Reply:
Yes, it should.
```suggestion
To access a specific record, append its recordid surrounded by parentheses to the dataset address.
```